core/src/main/java/org/apache/spark/shuffle/sort/BypassMergeSortShuffleWriter.java

```diff
@@ -199,6 +214,14 @@ public long[] getPartitionLengths() {
     return partitionLengths;
   }
 
+  public RowBasedChecksum[] getRowBasedChecksums() {
```

Comment:
NIT: a comment to say it is for testing only.

Reply:
Done